sw/device/lib/drivers/rv_timer/rv_timer_regs.h

```diff
@@ -14,7 +14,7 @@
 extern "C" {
 #endif
 // Number of harts
-#define RV_TIMER_PARAM_N_HARTS 1
+#define RV_TIMER_PARAM_N_HARTS 2
```

Comment:
should't the rv_timer functions also be extended with the HART? if they are not already

Reply:
perfect

Reply:
Solved.